testinfra/modules/package.py

```diff
@@ -146,3 +149,20 @@ def version(self):
     def release(self):
         return self.check_output('rpm -q --queryformat="%%{RELEASE}" %s',
                                  self.name)
+
+
+class AlpinePackage(Package):
+
+    @property
+    def is_installed(self):
+        return self.check_output("apk -e info %s", self.name) == self.name
```

Comment:
I think this doesn't works for a non installed package since `apk` will exit with a status code 1 and will cause `check_output()` to raise an assertion.

I think this should be `return self.run_test('apk info -e %s', self.name).rc == 0`

Reply:
is_installed function really was not done correctly